### ISSUE TITLE
[doc] incorrect example for using User::pwent in the generic (get|set)pw* documentation

### ISSUE THREAD

**Issue by djerius** [Perl/perl5#21137]
**Where**

perlfunc

**Description**

At the bottom of documentation for `getpwnam` and pals (more easily obtained via `perldoc -f getpwnam`) is an example usage of `User::pwent`

```
use File::stat;
use User::pwent;
my $is_his = (stat($filename)->uid == pwent($whoever)->uid);
```

This is broken, as `pwent` isn't a function. Perhaps replace it with

```
my $is_theirs = (stat($filename)->uid == getpwnam($username)->uid);
```

which additionally makes the line gender neutral.

**Comment by rwp0** [comment]
Thanks @djerius you can submit a PR on `perlfunc.pod` file to fix such errors:

I've created the pull request below referencing this issue:

- https://github.com/jadeallenx/Pod-Perldoc/pull/48

---

Looks like we need to target the CPAN repository instead of Core:

- https://github.com/jadeallenx/Pod-Perldoc/blob/master/corpus/perlfunc.pod
- https://github.com/Perl/perl5/blob/blead/cpan/Pod-Perldoc/corpus/perlfunc.pod
- https://metacpan.org/pod/Pod::Perldoc

**Comment by haarg** [comment]
The `perlfunc.pod` file included with Pod::Perldoc is only for its own internal testing.

The correct file to patch is `pod/perlfunc.pod` in this repo.

**Comment by rwp0** [comment]
> The `perlfunc.pod` file included with Pod::Perldoc is only for its own internal testing.
> 
> The correct file to patch is `pod/perlfunc.pod` in this repo.

OK, thanks for letting us know. I created the PR below:

- https://github.com/Perl/perl5/pull/21145

Would be good to have that merged before 5.38 to avoid one incorrect example in Core 🙂

**Comment by djerius** [comment]
Thanks!
